docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -0,0 +1,710 @@
+          ["10s", 2],
+          ["1m", 2],
+          ["60s", 2],
+          ["5m", 2],
+          ["60m", 4]
+        ],
+        "overdue": 0,
+        "estimated_schedule_density": [0, 1, 0, 0, 0, 1, 0, 1, 0, 1, 0, 0, 0, 1, 0, 0, 1, 1, 1, 0, 0, 3, 0, 0, 0, 1, 0, 1, 0, 1, 0, 0, 0, 1, 0, 0, 1, 1, 1, 0]
+      },
+      "status": "OK"
+    }
+  }
+}
+--------------------------------------------------
+
+By analyzing the different sections of the output we can evaluate the variety of different theories that might explain the _drift_ identified in a deplyment.
+
+[[task-manager-health-evaluate-the-configuration]]
+===== Evaluate the Configuration
+
+[[task-manager-theory-reduced-polling-rate]]
+*Theory*:
+Perhaps {kib} is configured to poll for tasks at a reduced rate?
+
+*Diagnosis*:
+Evaluating the health stats above, we can see the following output under `stats.configuration.value`:
+
+[source,json]
+--------------------------------------------------
+{
+  "request_capacity": 1000,
+  "max_poll_inactivity_cycles": 10,
+  "monitored_aggregated_stats_refresh_rate": 60000,
+  "monitored_stats_running_average_window": 50,
+  "monitored_task_execution_thresholds": {
+    "default": {
+      "error_threshold": 90,
+      "warn_threshold": 80
+    },
+    "custom": {}
+  },
+  "poll_interval": 3000, # <1>
+  "max_workers": 10 # <2>
+}
+--------------------------------------------------
+<1> the `poll_interval` setting is configured to the default value of 3000 milliseconds
+<2> the `max_workers` setting is configured to the default value of 10 workers
+
+We can infer from this output that the {kib} instance is polling for work every 3 seconds and has the capacity to run 10 concurrent tasks.
+
+Hypothetically, lets suppose the output under `stats.configuration.value` was the following:
+
+[source,json]
+--------------------------------------------------
+{
+  "request_capacity": 1000,
+  "max_poll_inactivity_cycles": 10,
+  "monitored_aggregated_stats_refresh_rate": 60000,
+  "monitored_stats_running_average_window": 50,
+  "monitored_task_execution_thresholds": {
+    "default": {
+      "error_threshold": 90,
+      "warn_threshold": 80
+    },
+    "custom": {}
+  },
+  "poll_interval": 60000, # <1>
+  "max_workers": 1 # <2>
+}
+--------------------------------------------------
+<1> the `poll_interval` setting is configured to a value of 60000 milliseconds, far higher than the default
+<2> the `max_workers` setting is configured to a value of 1 worker, far lower than the default
+
+We can infer from this output that the {kib} instance is only polling for work once a minute and even then, it will only pick up one task at a time. This throughput is unlikely to support mission critical services such as Alerting or Reporting, as it means tasks will usually run late.
+
+There are two possible reasons for such a configuration:
+
+The first, is that these settings have been configured manually, which can be resolved by reconfiguring these settings.
```

Comment:
I tried that at first, but it became hard to read because the paragraph and code snippet after belong to the second bullet, and it reads weirdly to me.

I've made the first sentence of that paragraph into a bullet, and the rest appears after the list.
Does that look right to you? I'm concerned it might be confusing, but I'm not sure. 🤷

Reply:
This is what I mean:

<img width="816" alt="Screenshot 2021-03-03 at 16 07 39" src="https://user-images.githubusercontent.com/386208/109834925-9aae6580-7c3a-11eb-8df5-ba58201a71e1.png">

The paragraph and log snippet belong inside the second bullet point, but now they feel like they're more general.
Any thoughts on how we can format this better? 🤔